examples/s3/setbucketpolicy.go

```diff
@@ -48,4 +48,11 @@ func main() {
 	if err != nil {
 		log.Fatalln(err)
 	}
+
```

Comment:
Create a new file we don't need to piggyback on bucket policies example.  Create an example `examples/s3/setbucketlifecycle.go`

Reply:
I've already done it. Sorry for confusion, please see latest commit 9c269de